pkg/mssqlcontainer/ingest/mechanism/bacpac.go

```diff
@@ -0,0 +1,75 @@
+package mechanism
+
+import (
+	"fmt"
+	"github.com/microsoft/go-sqlcmd/internal/container"
+)
+
+type bacpac struct {
+	controller  *container.Controller
+	containerId string
+}
+
+func (m *bacpac) Initialize(controller *container.Controller) {
+	m.controller = controller
+}
+
+func (m *bacpac) CopyToLocation() string {
+	return "/var/opt/mssql/backup"
+}
+
+func (m *bacpac) Name() string {
+	return "dacfx"
+}
+
+func (m *bacpac) FileTypes() []string {
+	return []string{"bacpac", "dacpac"}
+}
+
+func (m *bacpac) BringOnline(databaseName string, containerId string, query func(string), options BringOnlineOptions) {
+	m.containerId = containerId
+	m.installSqlPackage()
+	m.setDefaultDatabaseToMaster(options.Username, query)
+
+	m.RunCommand([]string{
+		"/opt/sqlpackage/sqlpackage",
+		"/Diagnostics:true",
+		"/Action:import",
+		"/SourceFile:" + m.CopyToLocation() + "/" + options.Filename,
+		"/TargetServerName:localhost",
+		"/TargetDatabaseName:" + databaseName,
+		"/TargetTrustServerCertificate:true",
+		"/TargetUser:" + options.Username,
+		"/TargetPassword:" + options.Password,
+	})
+}
+
+func (m *bacpac) setDefaultDatabaseToMaster(username string, query func(string)) {
+	alterDefaultDb := fmt.Sprintf(
+		"ALTER LOGIN [%s] WITH DEFAULT_DATABASE = [%s]",
+		username,
+		"master")
+	query(alterDefaultDb)
+}
+
+func (m *bacpac) installSqlPackage() {
+	if m.controller == nil {
+		panic("controller is nil")
+	}
+
+	m.controller.DownloadFile(
+		m.containerId,
+		"https://aka.ms/sqlpackage-linux",
```

Comment:
>s/sqlpackage-linux", [](http://example.com/codeflow?start=16&length=20)

is sqlpackage CPU architecture agnostic? Does this also bring down Net Core runtime?